app/views/submissions/_form.html.erb

```diff
@@ -63,8 +70,61 @@
 					<%= t('submission.no_manuscript_yet') %>
 				</span>
 			</div>
-	    <div class="col-12" id="userManuscript-informer">
-  	  </div>
+	    <div class="col-12 mt-2" id="userManuscript-status">
+			<div class="col-12" id="file-size-correct" class="submission-initially-hidden">
+				<i class="fas fa-check"></i>
+				<%=t('submission.correct_file_size')%>
+			</div>
+			<div class="col-12" id="file-size-too-big" class="submission-initially-hidden">
+				<i class="fas fa-exclamation-triangle"></i>
+				<%=t('submission.file_size_too_big')%>
+			</div>
+			<div class="col-12" id="file-size-way-too-big" class="submission-initially-hidden">
+				<i class="fas fa-bomb"></i>
+				<%=t('submission.file_size_way_too_big')%>
+			</div>
+			<div class="col-12" id="file-optimize" class="submission-initially-hidden">
+				<small>
+					<%= t("submission.optimization_help_html")%>
+					<%= helpdesk(t('submission.optimization_info'),true)%>
+				</small>
+				<div class="col-12 my-1">
+					<button id="userManuscript-optimize-btn"
+						class="btn btn-sm btn-primary"
+						data-tr-working="<%= t('submission.working')%>"
+						data-tr-failed="<%= t('submission.optimization_failed')%>"
+						>
+						<%= t('submission.optimize') %>
+					</button>
+				</div>
+			<!--<div class="col-1"></div>-->
+				<div class="col-11">
+					<b id="log-btn"><%=t('submission.optimization_log')%></b>
+					<div id="userManuscript-optimize-log" class="submission-initially-hidden">
+					</div>
```

Comment:
There seems to be a bug (which I do not understand as the code seems perfectly fine): `#userManuscript-optimize-log` does not pick up `.submission-initially-hidden` so it is visible from the beginning and the logs are displayed. It works if you add `display: none;` here, don't know why.

As i played around with that I also came up with some design suggestion:
```
<span id="log-btn" class="badge badge-light clickable">
  <%=t('submission.optimization_log')%>
</span>
div id="userManuscript-optimize-log" class="submission-initially-hidden border bg-light p-2">
</div>
```